### ISSUE TITLE
lists, mutuals only, and eliminating the need for multiple accounts.

### ISSUE THREAD

**Issue by hoodiek** [tootsuite/mastodon#422]
i would advocate for making the posting options a drop down or a toggle to show a list of choices. users could then create and maintain a list of people to post to. this removes the need for multiple accounts, and makes a couple of preset options available as well, namely:

public (default)
unlisted
mutuals
(custom lists that are a subset of followers would follow)

this could also allow a seperate check box to make a toot unboostable. i understand this is a dramatic change, but this is a powerful one to give users. thank you for listening

**Comment by nikkisquared** [comment]
I think these are all good ideas, however, I want to add that I don't think this removes the need for multiple accounts for everybody. At that, I don't think "solving" multiple accounts is necessarily a good thing!

I want to try to make myself clear here. I would use all of these options if they were implemented. However, I would still maintain a separate, very private account whenever I decide I want that on Mastodon. A separate account for me means how I use it is so different from my main account, that it's just incompatible to come from the same source. To me, managing multiple accounts is essentially a higher level of managing custom lists for who can see what.

Another reason I care about this is because of how easy it could be to choose the wrong settings when tooting something. Already I've accidentally posted things on the public timeline I meant not to.  It's been minor so far but, say if I had a very private message that I chose the wrong list of recipients for, it could be very embarrassing. Separate accounts means it's a lot harder to post a toot to the wrong people.

Again, I think this issue is a good list of things to add. I just don't want to see potential support for users with multiple accounts be ignored by trying to "solve" it.

**Comment by hoodiek** [comment]
these are very good points. frankly these are in fact aimed at my own needs. however i also am taking into account that the dev said (at least for now) that the multiple-account support like tweetdeck is a wont-fix for the default-client, so i addressed the best that could be achieved in client now that i could imagine, to meet my own needs

**Comment by marrus-sh** [comment]
Supporting custom lists probably depends on #134, and I would imagine need not be limited only to followers, but obviously non-followers would not see the toots unless they seeked them out

**Comment by LogicalDash** [comment]
If we get custom lists, I'd like a convenient way to toot at a list consisting of one specific person. This would serve adequately for direct messages.

Should the recipient of a targeted toot be informed of its targeted nature, beyond its lack of a boost button?

**Comment by hoodiek** [comment]
i dont think that's really necessary, but i may be the minority in that opinion

**Comment by nikkisquared** [comment]
I think it would be good to know what the scope of a toot directed at one person is, so you know what the intended privacy is. Like, if you send somebody a direct message on twitter, that's because you're saying "this is only for you". If there's way to tell the difference between a toot that's been made private, for followers only, and a toot that's private for just you, I think there's potential for the recipient to accidentally make a public response to it.

**Comment by hoodiek** [comment]
that's slightly covered in another issue i've made, but that is a valid point. i think that that would fit in the context of a toot better than immediate-in-column visibility

**Comment by hoodiek** [comment]
see #440 for the issue of levels of confidentiality in posting

**Comment by Lana-chan** [comment]
updates on this?

**Comment by kaniini** [comment]
mutuals only setting would be a nice enhancement.  additionally the ability to restrict posts to your own instance.

**Comment by Cassolotl** [comment]
@kaniini 

>additionally the ability to restrict posts to your own instance.

You might find #861 good to subscribe to and +1!

**Comment by Cassolotl** [comment]
My feelings on this issue are that the current privacy settings are really confusing. Everything is set up oddly.

Posting things for followers-only is nice, but we have no way of controlling who follows us aside from just locking everything down and being invisible, which is really not great at all. When I've brought this up before here on Github, I'm told to make my account private so I have to approve follow requests, but that doesn't really resolve the issue because I'm quite happy having a public account and letting anyone follow me.

So, to make things less confusing, I (in my short-sighted kind of way) would change "followers only" to "mutuals only" - and I would move DMs out of timelines and notifications and into a different section somehow, because it frightens me to see something very private jumbled among public posts by everyone.

**Comment by endlist** [comment]
@Cassolotl Yeah this is my issue also.  I make half my posts public and want people to be able to follow/interact with my public posts while also being able to make private posts just for friends I choose to allow.  Honestly it'd be nice if it wasn't tied to following at all, because I might want to follow someone and still not have them see my private posts even if they follow me also.  Basically I kind of want Dreamwidth subscribe/access split but for a twitter-like interface.

**Comment by Cassolotl** [comment]
@endlist

>Honestly it'd be nice if it wasn't tied to following at all, because I might want to follow someone and still not have them see my private posts even if they follow me also.

Oh yes! Fair enough. I think I could live without that but I totally see where you're coming from, there.

**Comment by Lana-chan** [comment]
has anyone involved in the mastodon project weighed in on this yet and why is the answer no

**Comment by alugarius** [comment]
I also want private messeges, adding people so I can use it like groupchats.

Please make that happen

**Comment by Cassolotl** [comment]
I ran into a need for this today. There's someone I don't mind following me, but also I want to toot privately and have them not see it.

Basically I want anyone to be able to follow me, but I want to be able to have a whitelist for who gets to see my private toots. If that is my mutuals, that'd be fine and cool with me!

**Comment by LogicalDash** [comment]
I have a single follower on Friendica, which I don't think will respect followers-only. On occasion I'd like to post something private and not have it federate to that one person.
